### PR TITLE
Phone Number add to result

### PR DESCRIPTION
**Pull request by rantifamilusi**
# Description

Add phone number to search
Generate class from open api spec
Refactored address to use generated class

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Locally unit test.
Test with Dynamics

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## For bcgov contributors:

this PR fixes jira ticket: **FAMS3-942, FAMS3-943, FAMS-944**

### CODE REVIEW

File: app/DynamicsAdapter/DynamicsAdapter.Web/searchApi.openapi.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "description": "For Search",
     "version": "V0.1"
   },
-  "host": "localhost:5050",
+  "host": "localhost:64834",
```
Comment (peggy-gov-bc-ca) [inline app/DynamicsAdapter/DynamicsAdapter.Web/searchApi.openapi.json]: do not know if this will work on openshift.

Comment (rantifamilusi) [reply app/DynamicsAdapter/DynamicsAdapter.Web/searchApi.openapi.json]: this is the spec to generate the class libraries.. actual configuration are in openshift and this is not in use.

File: app/DynamicsAdapter/Fams3Adapter.Dynamics/Types/TelephoneNumberType.cs [hunk 2]
```diff
@@ -12,6 +12,7 @@ public class TelephoneNumberType : Enumeration
         public static TelephoneNumberType Home = new TelephoneNumberType(867670001, "Home");
         protected TelephoneNumberType(int value, string name) : base(value, name)
```
Comment (peggy-gov-bc-ca) [inline app/DynamicsAdapter/Fams3Adapter.Dynamics/Types/TelephoneNumberType.cs]: only 1 type?

Comment (rantifamilusi) [reply app/DynamicsAdapter/Fams3Adapter.Dynamics/Types/TelephoneNumberType.cs]: There are more than one See actual file for detail. That must have been an update that was seen as a change

File: app/DynamicsAdapter/DynamicsAdapter.Web/searchApi.openapi.json (hunk 1 above)
Comment (peggy-gov-bc-ca) [reply app/DynamicsAdapter/DynamicsAdapter.Web/searchApi.openapi.json]: ok.

**Pull request merged**